docs/v23/configuration/job-router-overview.md

```diff
@@ -89,6 +89,23 @@ Additionally, it is now easier to include job transformations that should be eva
 including transforms in the lists of `JOB_ROUTER_PRE_ROUTE_TRANSFORM_NAMES` and `JOB_ROUTER_PRE_ROUTE_TRANSFORM_NAMES`,
 respectively.
 
+### Converting To New Syntax ###
+
+For existing HTCondor-CE's utilizing the deprecated old job router syntax can do the following steps to convert
+to using the new syntax:
+
+1.  Output the current configuration by running `condor_ce_config_val -summary > summary-file`
+2.  Convert the stored configuration by running `condor_transform_ads -convert:file summary-file`
```

Comment:
Does condor_transform_ads print the result to stdout or create a file? If it's the former, you should add a sample redirect to the command such as `condor_transform_ads -convert:file summary-file > transformed-config.conf`

Reply:
Do we want `transformed-config.conf` or something else like `converted-job-routes.conf`?